multi_import/_txt.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from tablib.compat import is_py3, StringIO
+
+
+title = 'txt'
+extensions = ('txt',)
+
+
+DEFAULT_ENCODING = 'utf-8'
+DEFAULT_DELIMITER = ','
+
+
+def export_set(dataset, **kwargs):
+    """Returns CSV representation of Dataset."""
+    stream = StringIO()
+
+    if not is_py3:
+        kwargs.setdefault('encoding', DEFAULT_ENCODING)
+
+    for row in dataset._package():
+        for key, val in row.iteritems():
+            stream.write('-' * len(key) + '\n')
+            stream.write(key.encode('utf-8') + '\n')
+            stream.write('-' * len(key) + '\n')
```

Comment:
yes, this was intentional so we get output like

`-------
task_id
-------
T1

------------
universal_id
------------
ff19485c-3607-48df-8269-5b0f0017c799
`

the * are for the section breaks.